guardian/transcription-service
# Add initial s3 infrastructure, zod types

## What does this change?
Paired with @marjisound and @zekehuntergreen 

This PR:
 - Adds a new S3 bucket to hold source media for the transcription service
 - Adds permissions to the worker to fetch and delete from the bucket
 - Updates packages/common/src/types.ts to include all relevant fields we think will be needed to be passed between our different services

## How to test
This code doesn't actually do anything yet so I think is safe to merge if the build passes. I've deployed to CODE to check that the deploy works
